src/EnvChecker.php

```diff
@@ -63,4 +61,42 @@ protected function checkUndocumented(array $meta): array
             $meta
         );
     }
+
+    protected function complain(array $errors): void
+    {
+        if (0 === sizeof($errors)) {
+            return;
+        }
+
+        $this->registerExceptionHandler();
+
+        throw new InvalidEnvException(implode('; ', $errors));
+    }
+
+    protected function registerExceptionHandler()
```

Comment:
This method registers an error handler which will output "Envoi validation failed" to the browser when the EnvCheckers throw InvalidEnvExceptions